packages/web-react/src/components/TextField/README.md

```diff
@@ -21,6 +21,7 @@ and show if the input is required.
 | `placeholder`     | string                        | Input placeholder              |
 | `value`           | string                        | Input value                    |
 | `message`         | string                        | Validation or help message     |
+| `size`            | number                        | Input width                    |
```

Comment:
Can we please use a different name for this prop? In our discussions about dictionaries, we use `Size` and `SizeExtended` dictionaries with values like `small`, `medium`, `large`, etc. So this name in this case is confusing when used without those values. The same problem is with the `size` prop in the Icon component.

Reply:
Ok, I suggest `inputWidth`. What do you think? @literat @adamkudrna

Reply:
I am ok with the `inputWidth` based on a personal discussion with @dlouhak.

Reply:
Added BC commit to the [React](https://github.com/lmc-eu/spirit-design-system/pull/538/commits/3a8a52ed5c24b8df2c1102c3a2accd1b0fb27667) and [Twig](https://github.com/lmc-eu/spirit-design-system/pull/537/commits/bc9eba976b36cf76c3b8714a05d7f6b935227356) module.

Reply:
👍